plugins/hnsw/package.json

```diff
@@ -0,0 +1,75 @@
+{
+  "name": "genkitx-hnsw",
+  "description": "Firebase Genkit AI framework plugin for HNSW vector database. Get AI response enriched with additional context and knowledge with HNSW Vector Database using RAG Implementation",
+  "keywords": [
+    "genkit-plugin",
+    "genkit-indexer",
+    "genkit-retriever",
+    "genkit-embedder",
+    "genkit-flow",
+    "genkit-rag",
+    "hnsw",
+    "vector",
+    "rag",
+    "ai",
+    "generative-ai",
+    "genai"
+  ],
+  "version": "0.1.5-rc.1",
+  "type": "commonjs",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/TheFireCo/genkit-plugins.git",
+    "directory": "plugins/hnsw"
+  },
+  "author": "TheFireCo",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@genkit-ai/flow": "^0.5.0",
+    "@genkit-ai/googleai": "^0.5.0",
+    "@langchain/google-genai": "^0.0.11",
+    "@types/node": "^20.12.12",
+    "@types/jest": "^29.5.12",
+    "fs": "^0.0.1-security",
+    "glob": "^10.4.1",
+    "hnswlib-node": "^1.3.0",
+    "langchain": "^0.0.11",
+    "node-fetch": "^3.2.6",
+    "redis": "^4.6.13",
+    "zod": "^3.22.4"
```

Comment:
@davidoort I moved `@genkit-ai/ai` and `@genkit-ai/core` to peer dependencies, not sure if some of these need to be moved there as well. Maybe worth checking.